python/cudf/cudf/tests/test_parquet.py

```diff
@@ -3776,10 +3776,10 @@ def test_parquet_chunked_reader(
     chunk_read_limit, pass_read_limit, use_pandas_metadata, row_groups
 ):
     df = pd.DataFrame(
-        {"a": [1, 2, 3, 4] * 1000000, "b": ["av", "qw", "hi", "xyz"] * 1000000}
+        {"a": [1, 2, 3, None] * 10000, "b": ["av", "qw", None, "xyz"] * 10000}
```

Comment:
Trivial: Reduced the number of rows to make this test run much quicker.

Reply:
Done in https://github.com/rapidsai/cudf/pull/17321/commits/241a88f39d20d68cf12a9f84e3ad4762dec27379